docs/kyma/docs/037-gs-installation-overrides.md

```diff
@@ -0,0 +1,168 @@
+Example:
+
+There's a `core` top-level chart, that is installed by the Installer.
+In one of it's templates there's a following line with a value reference:
+```
+memory: {{ .Values.test.acceptance.ui.requests.memory }}
+```
+In order to resolve this template, there's a default value for "test.acceptance.ui.requests.memory" in the `values.yaml` file of the chart.
+The following fragment of `values.yaml` shows this definition:
+```
+test:
+  acceptance:
+    ui:
+      requests:
+        memory: "1Gi"
+```
+
+If you want to override this value, for example to "2Gi", proceed as follows:
+- Create a ConfigMap in `kyma-installer` namespace, labelled with: `installer:overrides` (or reuse an existing one)
+- Add an entry `test.acceptance.ui.requests.memory: 2Gi` to the map.
+
+Once the installation starts, Installer generates overrides based on the map entries and value of "2Gi" will be used instead of default "1Gi" from the chart `values.yaml` file.
+For values that should be kept in Secrets, just define a Secret object instead of ConfigMap with the same key and value. Don't forget to label the Secret with `installer:overrides` and remember that values in Secrets must be base64-encoded.
+
+
+#### Sub-chart overrides
+
+Overrides for sub charts follow the same convention as top-level charts, but require additional information about sub-chart location.
+When a sub-chart contains `values.yaml` file, the information about chart location is not necessary because chart and it's `values.yaml` file are on the same level in directory hierarchy.
+The situation is different when the Installer installs a chart with sub-charts.
+All template values for a sub-chart must be prefixed with a sub-chart "path" (relative to top-level "parent" chart).
+This in not Installer-specific requirement, the same considerations apply when you provide overrides manually via `helm` command-line tool.
+
+Example:
+
+There's a `core` top-level chart, that is installed by the Installer.
+There's an `application-connector` sub-chart in `core` with another nested sub-chart: `connector-service`.
+In one of it's templates there's a following fragment (shortened):
+```
+spec:
+  containers:
+  - name: {{ .Chart.Name }}
+	args:
+	  - "/connectorservice"
+	  - '--appName={{ .Chart.Name }}'
+	  - "--domainName={{ .Values.global.domainName }}"
+	  - "--tokenExpirationMinutes={{ .Values.deployment.args.tokenExpirationMinutes }}"
+```
+
+The following fragment of `values.yaml` file in `connector-service` chart defines default value for `tokenExpirationMinutes`:
+```
+deployment:
+  args:
+    tokenExpirationMinutes: 60
+```
+
+If you want to override this value, for example to "90", do as follows:
+- Create a ConfigMap in kyma-installer namespace, labelled with: `installer:overrides` (or reuse existing one)
+- Add an entry `application-connector.connector-service.deployment.args.tokenExpirationMinutes: 90` to the map.
+
+Notice that the user-provided override key is now composed from two parts:
```

Comment:
Notice that the user-provided override key now contains two parts:
  - The chart "path" inside the top-level `core` chart `application-connector.connector-service`
  - The original template value reference from the chart without the `.Values.` prefix  `deployment.args.tokenExpirationMinutes`.

Once the installation starts, the Installer generates overrides based on the map entries. The system uses the value of "90" instead of the default value of "60" from the `values.yaml` chart file.